ciao-launcher/tests/ciao-launcher-server/server.go

```diff
@@ -491,6 +495,8 @@ func createConfigFile(confPath string) error {
 	conf.Configure.Launcher.MemoryLimit = memLimit
 	conf.Configure.Launcher.ComputeNetwork = []string{computeNet}
 	conf.Configure.Launcher.ManagementNetwork = []string{mgmtNet}
+	conf.Configure.Storage.SecretPath = secretPath
```

Comment:
Actually if I add the default value in the flag, wouldn't I always want to be doing the assignment since it would contain either a value passed by the user or the default value?